Pipfile

```diff
@@ -4,23 +4,24 @@ verify_ssl = true
 name = "pypi"
 
 [dev-packages]
-django-debug-toolbar = "==1.9.1"
-django-extensions = "*"
+django-debug-toolbar = "==3.1.1"
+django-extensions = "==3.0.9"
 
 [packages]
-django-dotenv = "==1.4.1"
-"psycopg2" = "==2.7.3.2"
-django-allauth = "==0.42.0"
+celery = "==5.0.2"
+django = "==3.1.3"
```

Comment:
Since this project/repository doesn't have many people actively working on it any type of frequent changes, it is preferable to stick to LTS releases (for `django` and for any package that might have it), this way updates can be applied quickly and don't require extra work to deal with any backwards incompatible changes.

Note: In this file there are other updates that jump some major versions, since the test suite currently cover a very small part of the functionality, we will have to manually make sure this doesn't break any functionality.